src/libraries/System.Console/ref/System.Console.cs

```diff
@@ -8,58 +8,81 @@ namespace System
-        public static System.IO.TextReader In { get { throw null; } }
-        public static System.Text.Encoding InputEncoding { get { throw null; } set { } }
+        public static System.IO.TextReader In { [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")] get { throw null; } }
+        public static System.Text.Encoding InputEncoding { [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")] get { throw null; } set { } }
         public static bool IsErrorRedirected { get { throw null; } }
         public static bool IsInputRedirected { get { throw null; } }
         public static bool IsOutputRedirected { get { throw null; } }
         public static bool KeyAvailable { get { throw null; } }
-        public static int LargestWindowHeight { get { throw null; } }
-        public static int LargestWindowWidth { get { throw null; } }
+        public static int LargestWindowHeight { [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")] get { throw null; } }
+        public static int LargestWindowWidth { [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")] get { throw null; } }
         [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows")]
         public static bool NumberLock { get { throw null; } }
         public static System.IO.TextWriter Out { get { throw null; } }
         public static System.Text.Encoding OutputEncoding { get { throw null; } set { } }
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
         public static string Title { [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows")] get { throw null; } set { } }
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
         public static bool TreatControlCAsInput { get { throw null; } set { } }
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
         public static int WindowHeight { get { throw null; } [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows")] set { } }
         public static int WindowLeft { get { throw null; } [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows")] set { } }
         public static int WindowTop { get { throw null; } [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows")] set { } }
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
         public static int WindowWidth { get { throw null; } [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows")] set { } }
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
         public static event System.ConsoleCancelEventHandler? CancelKeyPress { add { } remove { } }
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
         public static void Beep() { }
         [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows")]
```

Comment:
Missing [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")] ?

Reply:
Excluded for the same reasoning as https://github.com/dotnet/runtime/pull/41184#discussion_r483085982